README.md

```diff
@@ -80,6 +80,14 @@ Options
         Allows you to use your own reflector
         Default: http://reflector2.datastax.com/reflector2.php
 
+    --repository <repository>:<branch>
```

Comment:
Standard separator to split on seems to be '/' instead of ':'. This is what hub (https://hub.github.com/) and bundler (http://hashnuke.com/2012/06/05/bundler-new-github-option.html)

I can fix this one post-merge.